### PR TITLE
Add Bitcoin Cash 

### PR DESCRIPTION
**Pull request by shilch**
**https://github.com/dsmurrell/awesome-bitcoin-cash**

**A curated list of awesome things related to Bitcoin Cash.**

I am not owner of the list and can not change things like e.g. the topics as it is required.  
So this is just a kick-start for @dsmurrell to add his list here.

### By submitting this pull request I confirm I've read and complied with the below requirements 🖖

**Please read it multiple times. I spent a lot of time on these guidelines and most people miss a lot.**

## Requirements for your pull request

- I have read and understood the [contribution guidelines](https://github.com/sindresorhus/awesome/blob/master/contributing.md) and the [instructions for creating a list](https://github.com/sindresorhus/awesome/blob/master/create-list.md).
- This pull request has a descriptive title.<br>For example, `Add Name of List`, not `Update readme.md` or `Add awesome list`.
- The entry in the Awesome list should:
	- Include a short description about the project/theme of the list. **It should not describe the list itself.**<br>Example: `- [Fish](…) - User-friendly shell.`, not `- [Fish](…) - Resources for Fish.`.
	- Be added at the bottom of the appropriate category.
- The list I'm submitting complies with these requirements:

## Requirements for your Awesome list

- **Has been around for at least 30 days.**<br>That means 30 days from either the first real commit or when it was open-sourced. Whatever is most recent.
- It's the result of hard work and the best I could possibly produce.
- Non-generated Markdown file in a GitHub repo.
- **Includes a succinct description of the project/theme at the top of the readme.** [(Example)](https://github.com/willempienaar/awesome-quantified-self)
- The repo should have `awesome-list` & `awesome` as [GitHub topics](https://help.github.com/articles/about-topics). I encourage you to add more relevant topics.
- Not a duplicate.
- Only has awesome items. Awesome lists are curations of the best, not everything.
- Includes a project logo/illustration whenever possible.
	- Either fullwidth or placed at the top-right of the readme. [(Example)](https://github.com/sindresorhus/awesome-electron)
	- The image should link to the project website or any relevant website.
	- The image should be high-DPI. Set it to maximum half the width of the original image.
- Entries have a description, unless the title is descriptive enough by itself. It rarely is though.
- Includes the [Awesome badge](https://github.com/sindresorhus/awesome/blob/master/awesome.md#awesome-badge).
	- Should be placed on the right side of the readme heading.
	- Should link back to this list.
- Has a Table of Contents section.
	- Should be named `Contents`, not `Table of Contents`.
	- Should be the first section in the list.
	- Should only have one level of sub-lists, preferably none.
- Has an [appropriate license](https://github.com/sindresorhus/awesome/blob/master/awesome.md#choose-an-appropriate-license).
	- That means something like CC0, **not a code licence like MIT, BSD, Apache, etc.**
	- [WTFPL](http://www.wtfpl.net) and [Unlicense](http://unlicense.org) are not acceptable licenses.
	- If you use a license badge, it should be SVG, not PNG.
- Has [contribution guidelines](https://github.com/sindresorhus/awesome/blob/master/awesome.md#include-contribution-guidelines).
	- The file should be named `contributing.md`. Casing is up to you.
- Has consistent formatting and proper spelling/grammar.
	- The link and description are separated by a dash. <br>Example: `- [AVA](…) - JavaScript test runner.`
	- The description starts with an uppercase character and ends with a period.
	- Consistent and correct naming. For example, `Node.js`, not `NodeJS` or `node.js`.
- Doesn't include a Travis badge.<br>You can still use Travis for list linting, but the badge has no value in the readme.

Go to the top and read it again.

### CODE REVIEW

**Comment by davisonio** [comment]
Requirements not followed above

**Comment by shilch** [comment]
Yeah @dsmurrell has to modify his list according to the requirements.

**Comment by dsmurrell** [comment]
@shilch thanks for adding this. I'll work through following the requirements when I can!

**Comment by sindresorhus** [comment]
Hey, sorry for the lack of response, please see #1363.

**Comment by sindresorhus** [comment]
```
❯ awesome-lint https://github.com/dsmurrell/awesome-bitcoin-cash
✖ Linting

  README.md:19:1
  ✖    1:1    Missing Awesome badge after the main heading                                 remark-lint:awesome/badge
  ✖    1:1    Missing License section                                                      remark-lint:awesome/license
  ✖    1:1    Missing or invalid Table of Contents                                         remark-lint:awesome/toc
  ✖    3:3    Link to https://rawgit.com/dsmurrell/awesome-bitcoin-cash is dead            remark-lint:no-dead-urls
  ✖   19:1    Don’t use multiple top level headings (3:1)                                  remark-lint:no-multiple-toplevel-headings
  ✖   21:1    Missing initial pipe in table fence                                          remark-lint:table-pipes
  ✖   21:87   Cell should be compact                                                       remark-lint:table-cell-padding
  ✖   21:114  Missing final pipe in table fence                                            remark-lint:table-pipes
  ✖   21:114  Cell should be compact                                                       remark-lint:table-cell-padding
  ✖   23:1    Missing initial pipe in table fence                                          remark-lint:table-pipes
  ✖   23:33   Misaligned table fence                                                       remark-lint:table-pipe-alignment
  ✖   23:35   Cell should be compact                                                       remark-lint:table-cell-padding
  ✖   23:64   Cell should be compact                                                       remark-lint:table-cell-padding
  ✖   23:65   Misaligned table fence                                                       remark-lint:table-pipe-alignment
  ✖   23:93   Misaligned table fence                                                       remark-lint:table-pipe-alignment
  ✖   23:95   Cell should be compact                                                       remark-lint:table-cell-padding
  ✖   23:121  Misaligned table fence                                                       remark-lint:table-pipe-alignment
  ✖   23:122  Cell should be compact                                                       remark-lint:table-cell-padding
  ✖   23:122  Missing final pipe in table fence                                            remark-lint:table-pipes
  ✖   24:1    Missing initial pipe in table fence                                          remark-lint:table-pipes
  ✖   24:49   Misaligned table fence                                                       remark-lint:table-pipe-alignment
  ✖   24:79   Misaligned table fence                                                       remark-lint:table-pipe-alignment
  ✖   24:105  Misaligned table fence                                                       remark-lint:table-pipe-alignment
  ✖   24:106  Missing final pipe in table fence                                            remark-lint:table-pipes
  ✖   25:1    Missing initial pipe in table fence                                          remark-lint:table-pipes
  ✖   25:43   Misaligned table fence                                                       remark-lint:table-pipe-alignment
  ✖   25:65   Misaligned table fence                                                       remark-lint:table-pipe-alignment
  ✖   25:91   Misaligned table fence                                                       remark-lint:table-pipe-alignment
  ✖   25:92   Missing final pipe in table fence                                            remark-lint:table-pipes
  ✖   26:1    Missing initial pipe in table fence                                          remark-lint:table-pipes
  ✖   26:48   Cell should be compact                                                       remark-lint:table-cell-padding
  ✖   26:49   Misaligned table fence                                                       remark-lint:table-pipe-alignment
  ✖   26:77   Misaligned table fence                                                       remark-lint:table-pipe-alignment
  ✖   26:97   Misaligned table fence                                                       remark-lint:table-pipe-alignment
  ✖   26:98   Missing final pipe in table fence                                            remark-lint:table-pipes
  ✖   27:1    Missing initial pipe in table fence                                          remark-lint:table-pipes
  ✖   27:21   Misaligned table fence                                                       remark-lint:table-pipe-alignment
  ✖   27:45   Misaligned table fence                                                       remark-lint:table-pipe-alignment
  ✖   27:47   Cell should be compact                                                       remark-lint:table-cell-padding
  ✖   27:75   Misaligned table fence                                                       remark-lint:table-pipe-alignment
  ✖   27:76   Cell should be compact                                                       remark-lint:table-cell-padding
  ✖   27:76   Missing final pipe in table fence                                            remark-lint:table-pipes
  ✖   28:1    Missing initial pipe in table fence                                          remark-lint:table-pipes
  ✖   28:48   Cell should be compact                                                       remark-lint:table-cell-padding
  ✖   28:49   Misaligned table fence                                                       remark-lint:table-pipe-alignment
  ✖   28:69   Misaligned table fence                                                       remark-lint:table-pipe-alignment
  ✖   28:93   Misaligned table fence                                                       remark-lint:table-pipe-alignment
  ✖   28:94   Missing final pipe in table fence                                            remark-lint:table-pipes
  ✖   29:1    Missing initial pipe in table fence                                          remark-lint:table-pipes
  ✖   29:37   Misaligned table fence                                                       remark-lint:table-pipe-alignment
  ✖   29:61   Misaligned table fence                                                       remark-lint:table-pipe-alignment
  ✖   29:86   Misaligned table fence                                                       remark-lint:table-pipe-alignment
  ✖   29:87   Missing final pipe in table fence                                            remark-lint:table-pipes
  ✖   30:1    Missing initial pipe in table fence                                          remark-lint:table-pipes
  ✖   30:27   Misaligned table fence                                                       remark-lint:table-pipe-alignment
  ✖   30:47   Misaligned table fence                                                       remark-lint:table-pipe-alignment
  ✖   30:48   Missing final pipe in table fence                                            remark-lint:table-pipes
  ✖   38:1    Don’t use multiple top level headings (3:1)                                  remark-lint:no-multiple-toplevel-headings
  ✖   57:3    Link to https://bitpay.com/tour is dead                                      remark-lint:no-dead-urls
  ✖   59:3    Link to https://rocketr.net is dead                                          remark-lint:no-dead-urls
  ✖   66:3    Link to https://www.minipos.cash/ is dead                                    remark-lint:no-dead-urls
  ✖   67:3    Link to https://pos.cheddr.cash/launch.html is dead                          remark-lint:no-dead-urls
  ✖   69:3    Link to https://moosh.io/wallet.html is dead                                 remark-lint:no-dead-urls
  ✖   70:3    Link to http://www.bitcoinc.com.au/spay is dead                              remark-lint:no-dead-urls
  ✖   75:3    Link to http://blocksize.party is dead                                       remark-lint:no-dead-urls
  ✖   83:3    Link to https://coinone.co.kr/exchange/trade/bch is dead                     remark-lint:no-dead-urls
  ✖   85:3    Link to https://bittrex.com/Market/Index?MarketName=BTC-BCC is dead          remark-lint:no-dead-urls
  ✖   87:3    Link to https://www.huobi.pro/exchange/bcc_btc is dead                       remark-lint:no-dead-urls
  ✖   88:3    Link to https://yunbi.com/markets/bcccny is dead                             remark-lint:no-dead-urls
  ✖   99:3    Link to https://bitcoin.vn/ is dead                                          remark-lint:no-dead-urls
  ✖  119:58   List item description must end with proper punctuation                       remark-lint:awesome/list-item
  ✖  127:3    Link to https://safedice.com is dead                                         remark-lint:no-dead-urls
  ✖  129:3    Link to https://mtbetmore.com/ is dead                                       remark-lint:no-dead-urls
  ✖  143:3    Link to https://www.blocktrail.com/BCC is dead                               remark-lint:no-dead-urls
  ✖  147:3    Link to https://bccblock.info is dead                                        remark-lint:no-dead-urls
  ✖  159:3    Link to https://cashvscore.com is dead                                       remark-lint:no-dead-urls
  ✖  165:3    Link to http://fork.watch is dead                                            remark-lint:no-dead-urls
  ✖  166:51   List item description must end with proper punctuation                       remark-lint:awesome/list-item
  ✖  167:37   List item description must end with proper punctuation                       remark-lint:awesome/list-item
  ✖  169:44   List item description must end with proper punctuation                       remark-lint:awesome/list-item
  ✖  181:1    Don’t use multiple top level headings (3:1)                                  remark-lint:no-multiple-toplevel-headings
  ✖  189:3    Link to https://tipcrypto.net is dead                                        remark-lint:no-dead-urls
  ✖  190:3    Link to https://blockpress.com is dead                                       remark-lint:no-dead-urls
  ✖  191:27   List item description must end with proper punctuation                       remark-lint:awesome/list-item
  ✖  210:3    Link to https://www.meetup.com/BitcoinCash-Medellin/ is dead                 remark-lint:no-dead-urls
  ✖  214:3    Link to https://www.meetup.com/Bitcoin-Cash-Cancun is dead                   remark-lint:no-dead-urls
  ✖  226:3    Link to https://www.meetup.com/St-Kilda-New-Technology-Meetup is dead        remark-lint:no-dead-urls
  ✖  227:3    Link to https://www.meetup.com/Bitcoin-Cash-in-Perth-Meetup/ is dead         remark-lint:no-dead-urls
  ✖  228:3    Link to https://www.meetup.com/Auckland-Bitcoin-Cash-Meetup/ is dead         remark-lint:no-dead-urls
  ✖  234:3    Link to https://www.meetup.com/Nizhny-Novgorod-Разработка-ПО-Meetup is dead  remark-lint:no-dead-urls
  ✖  235:3    Link to https://www.meetup.com/Bitcoin-cash-local-meetup/ is dead            remark-lint:no-dead-urls
  ✖  238:3    Link to https://www.meetup.com/Kuala-Lumpur-Bitcoin-Cash-Meetup/ is dead     remark-lint:no-dead-urls
  ✖  259:3    Link to https://www.meetup.com/Denver-Bitcoin-Cash-Meetup/ is dead           remark-lint:no-dead-urls
  ✖  260:3    Link to https://www.meetup.com/BitcoinCashGulfCoast/ is dead                 remark-lint:no-dead-urls
  ✖  262:3    Link to https://www.meetup.com/Basseterre-Bitcoin-Meetup/ is dead            remark-lint:no-dead-urls
  ✖  264:3    Link to https://www.meetup.com/Bitcoin-Cash-Florida-Meetup/ is dead          remark-lint:no-dead-urls
  ✖  267:3    Link to https://www.meetup.com/Las-Vegas-Henderson-Bitcoin-Meetup is dead    remark-lint:no-dead-urls
  ✖  268:3    Link to https://www.meetup.com/Seattle-Bitcoin-Cash-Meetup/ is dead          remark-lint:no-dead-urls
  ✖  296:1    Don’t use multiple top level headings (3:1)                                  remark-lint:no-multiple-toplevel-headings
  ✖  308:3    Link to https://bitpay.com/wallet is dead                                    remark-lint:no-dead-urls
  ✖  317:3    Link to https://bitpay.com/wallet is dead                                    remark-lint:no-dead-urls
  ✖  323:3    Link to https://www.unitwallet.co is dead                                    remark-lint:no-dead-urls
  ✖  342:3    Link to https://tools.bitcoin.com/paper-wallet/ is dead                      remark-lint:no-dead-urls
  ✖  349:1    Don’t use multiple top level headings (3:1)                                  remark-lint:no-multiple-toplevel-headings
  ✖  361:71   List item description must end with proper punctuation                       remark-lint:awesome/list-item
  ✖  363:36   List item description must end with proper punctuation                       remark-lint:awesome/list-item
  ✖  364:45   List item description must end with proper punctuation                       remark-lint:awesome/list-item
  ✖  365:43   List item description must end with proper punctuation                       remark-lint:awesome/list-item
  ✖  366:35   List item description must end with proper punctuation                       remark-lint:awesome/list-item
  ✖  367:3    Link to https://cryptohabit.com/ is dead                                     remark-lint:no-dead-urls
  ✖  367:42   List item description must end with proper punctuation                       remark-lint:awesome/list-item
  ✖  368:39   List item description must end with proper punctuation                       remark-lint:awesome/list-item
  ✖  369:44   List item description must end with proper punctuation                       remark-lint:awesome/list-item
  ✖  370:3    Link to https://greenhorncrypto.com/ is dead                                 remark-lint:no-dead-urls
  ✖  370:51   List item description must end with proper punctuation                       remark-lint:awesome/list-item
  ✖  371:40   List item description must end with proper punctuation                       remark-lint:awesome/list-item
  ✖  372:75   List item description must end with proper punctuation                       remark-lint:awesome/list-item
  ✖  373:3    Link to https://afrigarments.com/ is dead                                    remark-lint:no-dead-urls
  ✖  373:44   List item description must end with proper punctuation                       remark-lint:awesome/list-item

  119 errors
```

https://github.com/sindresorhus/awesome-lint

**Comment by sindresorhus** [comment]
> Includes a project logo/illustration whenever possible.
> **The image should link to the project website or any relevant website.**
> **The image should be high-DPI. Set it to maximum half the width of the original image.**

File: readme.md [hunk 1]
```diff
@@ -479,6 +479,7 @@
 ## Decentralized Systems
 
 - [Bitcoin](https://github.com/igorbarinov/awesome-bitcoin) - Bitcoin services and tools for software developers.
+- [Bitcoin Cash](https://github.com/dsmurrell/awesome-bitcoin-cash) - Bitcoin Cash resources.
```
Comment (sindresorhus) [inline readme.md]: > The entry in the Awesome list should:
* Include a short description about the project/theme of the list. **It should not describe the list itself.**

**Comment by sindresorhus** [comment]
Can you fix the merge conflict?

**Comment by sindresorhus** [comment]
Closing for lack of response.

**Pull request closed without merging**